pkg/kube/snapshot/snapshot_alpha.go

```diff
@@ -422,6 +422,19 @@ func UnstructuredVolumeSnapshotContentAlpha(name, snapshotName, snapshotNs, dele
 }
 
 func UnstructuredVolumeSnapshotClassAlpha(name, driver, deletionPolicy string, params map[string]string) *unstructured.Unstructured {
+	if params == nil {
+		return &unstructured.Unstructured{
+			Object: map[string]interface{}{
+				"apiVersion": fmt.Sprintf("%s/%s", v1alpha1.GroupName, v1alpha1.Version),
+				"kind":       VolSnapClassKind,
+				"metadata": map[string]interface{}{
+					"name": name,
+				},
+				VolSnapClassAlphaDriverKey: driver,
+				"deletionPolicy":           deletionPolicy,
+			},
+		}
+	}
```

Comment:
@viveksinghggits - There is a lot of common code in the map being constructed. Can we refactor this so that we prepare the map without the "parameters" first and then add the parameters if params is non nil ?